zanata-model/src/main/java/org/zanata/model/HTextFlowTarget.java

```diff
@@ -117,10 +118,25 @@
 
     private List<HTextFlowTargetReviewComment> reviewComments;
 
+    @Getter
+    private String revisionComment;
+
+    private boolean revisionCommentChanged = false;
+
     // Only for internal use (persistence transient)
     @Setter(AccessLevel.PRIVATE)
     private Integer oldVersionNum;
 
+    public void setRevisionComment(String revisionComment) {
+        this.revisionComment = revisionComment;
+        revisionCommentChanged = true;
```

Comment:
I don't see any difference between changing the `set` word in the method name. The general Java `setter` method doesn't limit to just set to different value, it's just a setter method of a value. The rules or limit of allowing to set same value or not should be sitting in business layer or the service class. 

In this case, same revisionComment value are allowed. Because it changes the behaviour of EntityListener for HTextFlowTarget when persisting.